mysql/datadog_checks/mysql/mysql.py

```diff
@@ -399,6 +408,57 @@ def _collect_replication_metrics(self, db, results, above_560):
         results.update(self._get_replica_status(db, above_560, nonblocking))
         return REPLICA_VARS
 
+    def _collect_group_replica_metrics(self, db, results):
+        try:
+            with closing(db.cursor()) as cursor:
```

Comment:
Since there is a lot of logic involved in collecting these metrics, it might make sense to hide group replication behind its own config flag.

Reply:
Let's keep a single config flag, I explicitly documented about the group replication support

Reply:
Is that in a separate PR?

Reply:
Just pushed :D